modules/navigation/godot_navigation_server.h

```diff
@@ -153,6 +153,12 @@ class GodotNavigationServer : public NavigationServer3D {
 
 	void flush_queries();
 	virtual void process(real_t p_delta_time) override;
+
+	virtual gd::PathQueryResult _query_path(const gd::PathQueryParameters &p_parameters) const override;
+
+private:
+	void _strip_path(const gd::PathQueryParameters &p_parameters, gd::PathQueryResult &p_query_result) const;
+	void _calc_path_length(const gd::PathQueryParameters &p_parameters, gd::PathQueryResult &p_query_result) const;
```

Comment:
I'm not entirely sure if this is a new convention in godot, but I think you can avoid prefixing with `_` even when the function is private. In other context prefix with `_` is used to signal that such thing is not used, also it seems just useless prefix the private function names as those are not used externally anyway.
However @akien-mga knows better than I do.

Reply:
No strong opinion here, there's a bit of everything in the codebase. `_` prefix for private methods is not uncommon and to me it makes sense. But it's also ok without.

Reply:
Personal preference to have helper functions that see only internal use with a underscore prefix. If it is no issue I would just keep it.